spec/fixtures/files/offlinevm.json

```diff
@@ -0,0 +1,55 @@
+{
+  "apiVersion": "kubevirt.io/v1alpha1",
+  "kind": "OfflineVirtualMachine",
+  "metadata": {
+    "generation": 124,
+    "creationTimestamp": "2018-01-24T10:15:25Z",
+    "name": "test",
+    "namespace": "default",
+    "resourceVersion": "262502",
+    "selfLink": "/apis/kubevirt.io/v1alpha1/namespaces/default/offlinevms/test",
+    "uid": "7e6fb1ac-00ef-11e8-8840-525400b2cba8",
+    "selector": {
```

Comment:
Selector is only on the preset, not on vm

Reply:
OK